### ISSUE TITLE
Lepidoptera collection with scientificName 'BiotaNA' for +1000 records

### ISSUE THREAD

**Issue by CecSve** [gbif/portal-feedback#4534]
The dataset [Lepidoptera Eastern Fennoscandia (Luomus) (LEPFEN)](https://www.gbif.org/dataset/0e631111-7b03-4894-8170-e3caeafe4bba) has +1000 records recorded as scientificName = BiotaNA and taxonRank superdomian which means the records are interpreted as the plant genus Biota ([example](https://www.gbif.org/occurrence/search?dataset_key=0e631111-7b03-4894-8170-e3caeafe4bba&taxon_key=6)). Some of the records have images of the collection material which is clearly Lepidoptera: https://www.gbif.org/occurrence/search?dataset_key=0e631111-7b03-4894-8170-e3caeafe4bba&media_type=StillImage&taxon_key=6 and those records would be best to publish as order = Lepidoptera (with the associated higher taxonomy). Since it is a dataset of Lepidoptera, it could be presumed all the BiotaNA records could be republished as Lepidoptera.

There is also an issue with missing higher taxonomy that causes +2000 records to be assigned as incertae sedis: https://www.gbif.org/occurrence/search?dataset_key=0e631111-7b03-4894-8170-e3caeafe4bba&taxon_key=0

Helpdesk will contact the data publisher and ask whether the records can be amended.

**Comment by CecSve** [comment]
Other datasets with BiotaNA in scientificName: https://www.gbif.org/occurrence/datasets?advanced=1&verbatim_scientific_name=BiotaNA&occurrence_status=present

**Comment by wkmor1** [comment]
Thanks for letting us know. The NA part of "BiotaNA" etc was a bug which has now been fixed (https://github.com/luomus/finbif/commit/b46b6a673a352f4cff779ea3099c74c011ef3299). Will contact the data provider re specifying lower taxa.

**Comment by CecSve** [comment]
@wkmor1 do you know when the archives will be updated? Then I can manually re-ingest and test whether interpretation works

**Comment by wkmor1** [comment]
Our archives update nightly. But there are ~150 of them with a total of ~40million occurrences so they update in random order running for 8hrs a night. It typically takes a week on average to update any given archive fully, but it can sometimes take up to a month. We trigger a reingest each time an archive is updated. If this is urgent I can try and manually update otherwise I'll just leave it to do its own thing.

**Comment by CecSve** [comment]
Thanks for the explanation! It is not urgent so I will make sure to keep track of the issue and close the issue once it is fixed. We are still [implementing a fix](https://github.com/gbif/pipelines/issues/850) to make sure that the records will be interpreted as incertae sedies and not a plant genus, so nu rush.

**Comment by CecSve** [comment]
Hi @wkmor1 - all the BiotaNA are resolved, but the higher taxonomy is still missing in ~4000 occurrences of the dataset: https://www.gbif.org/occurrence/search?dataset_key=0e631111-7b03-4894-8170-e3caeafe4bba&taxon_key=0 causing the occurrences to show up as incertae sedis. If possible, could the higher taxonomy be added to those records, e.g.: Class = Insecta, Order = Lepidoptera?

**Comment by wkmor1** [comment]
Yeah. Thanks for the reminder. The ones with no ID at all (about 1700 of them) are from errors a lot earlier in the digitization process. They are being fixed now and will flow through to GBIF in the next few weeks. The rest are from taxon names that aren't matched either by our own taxonomic backbone or GBIFs. I'll have to think about how to handle these. I'd rather not add data like higher taxonomy at this point in the process. It would be fine for this collection, as it obviously only has Lepidoptera in it. But it would not be great for other collections. Ideally the data owners/digitizers would add higher taxonomic information and it could just flow though in such cases. But unfortunately I doubt that will ever happen.

**Comment by CecSve** [comment]
Thank you for following up. I am a bit puzzled that so many names are not matching, though. @mdoering I might need you help here. There are a lot of incertae sedis records (4K) in [this dataset](https://www.gbif.org/dataset/0e631111-7b03-4894-8170-e3caeafe4bba) which I though was due to missing higher taxonomy. But when I run the unique names through the [species matching tool](https://www.gbif.org/tools/species-lookup) I get (fuzzy) matches for most and assigned higher taxonomy. 
![billede](https://github.com/gbif/portal-feedback/assets/54532645/9dc85973-e0ca-41a9-af05-ad25f8148d3f)
I thought this would mean it should work upon interpretation as well... but am I missing something?
